example/src/main/java/cloud/eppo/androidexample/EppoApplication.java

```diff
@@ -5,10 +5,11 @@
 
 import cloud.eppo.android.EppoClient;
 import cloud.eppo.android.InitializationCallback;
+import com.geteppo.androidexample.BuildConfig;
 
 public class EppoApplication extends Application {
     private static final String TAG = EppoApplication.class.getSimpleName();
-    private static final String API_KEY = "REPLACE WITH YOUR API KEY";
+    private static final String API_KEY = BuildConfig.API_KEY; // Set in root-level local.properties
```

Comment:
Pull the API key from `.gitignore`ed `local.properties` rather than having it here, where somebody could accidentally check one in.